scripts/zk_benchmark.py

```diff
@@ -0,0 +1,290 @@
+    
+    def ensure_root_node(self):
+        self.create(self.root_node)
+    
+    def clean(self, path: str):
+        self.zk.delete(path, recursive=True)
+    
+    def clean_all(self):
+        print("cleanup all in {}".format(self.root_node))
+        self.clean(self.root_node)
+
+# Data loader which loads data into Zk in chunked manner
+class DataLoader(object):
+    def __init__(self, config: DataLoaderConfig, zk_config: ZkClientConfig):
+        self.root_node = zk_config.root_node
+        self.config = config
+        self.zk_client = ZkClientWrapper(zk_config, "data_loader_zk")
+        
+    def __enter__(self):
+        self.zk_client.start()
+        return self
+    
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.zk_client.stop()
+        return True
+
+    def __get_chunked_generators(self, child_node_config: ChildNodeConfig) -> list[tuple[NodeGenerator, int]]:
+        rem_size = child_node_config.number
+        result = []
+        while rem_size > 0:
+            gen_size = min(rem_size, self.config.chunk_size)
+            result.append((node_generator(
+                gen_size, 
+                name=child_node_config.fixed_name, 
+                min_name_len=child_node_config.name_len, 
+                max_name_len=child_node_config.name_len+1, 
+                data_size_bytes=child_node_config.data_bytes), gen_size))
+            rem_size -= gen_size
+        
+        return result
+
+    def load_data(self, parent: str, child_node_config: ChildNodeConfig):
+        parent_node="{}/{}".format(self.root_node, parent)
+        self.zk_client.create(parent_node)
+
+        generators_with_size = self.__get_chunked_generators(child_node_config)
+        with concurrent.futures.ThreadPoolExecutor(max_workers=self.config.parallelism) as exec:
+            futures_to_id = {exec.submit(self._fill, parent_node, generator, size): id for (id, (generator, size)) in enumerate(generators_with_size)}
+            for future in concurrent.futures.as_completed(futures_to_id):
+                id = futures_to_id[future]
+                try:
+                    res = future.result()
+                except Exception as exc:
+                    print("loader: {} generated an exception: {}".format(id, exc))
+
+    def _fill(self, path: str, child_generator: NodeGenerator, gen_size: int):
+        for node in tqdm(child_generator, total=gen_size):
+            child_node = "{}/{}".format(path, node[0])
+            self.zk_client.create(child_node, data=node[1])
+
+class MultiProcessDataLoader:
+    def __init__(self, config: DataLoaderConfig, zk_config: ZkClientConfig):
+        self.root_node = zk_config.root_node
+        self.config = config
+        self.zk_config = zk_config
+    
+    def __enter__(self):
+        return self
+    
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        return True
+
+    def __get_chunked_configs(self, child_node_config: ChildNodeConfig) -> list[ChildNodeConfig]:
```

Comment:
with default chunk_size of 10K, and target range of 10K, this will effectively generate single block ?
should default be smaller number say 1K ?